src/pages/FAQ.tsx

```diff
@@ -0,0 +1,182 @@
+import { useQuery } from 'react-query';
+import { useState, useEffect } from 'react';
+import { Text, VStack, HStack, Button} from 'native-base';
+import { getFAQs, setFAQs } from '../xplat/queries/faq';
+import { FAQCollection, FAQItem } from '../xplat/types/faq';
+import FAQCardBox, { EditFAQCardBox } from '../components/FAQCardBox';
+
+const FAQ = () => {
+  const {data, isError, isLoading} = useQuery( 'faq_collection', () => getFAQs());
+  if (isError)
+  {
+    return (
+      <VStack alignItems='center' space={2}>
+        <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+        <Text>Error loading data.</Text>
+      </VStack>
+    );
+  }
+
+  if (isLoading || data === undefined)
+  {
+    return (
+      <VStack alignItems='center' space={2}>
+        <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      
+        <Text>Loading...</Text>
+
+      </VStack>
+    );
+  }
+
+  return (
+    <VStack alignItems='center' space={2}>
+      <img
+        src={process.env.PUBLIC_URL + '/logo.jpg'}
+        style={{
+          maxWidth: '200px',
+          height: 'auto'
+        }}
+        alt='logo'
+      />
+      <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      
+      {
+        data.map( (value, index) => 
+          <FAQCardBox key={index} question={value.question} answer={value.answer}/>)
+      }
+
+    </VStack>
+  );
+};
+
+export const EditFAQ = () => {
+  const [FAQCards, setFAQCards] = useState<FAQCollection>();
```

Comment:
should this default to an empty list of FAQItem?

Reply:
Very true, good catch